### ISSUE TITLE
Question about using rewards ref case:281

### ISSUE THREAD

**Issue by Equity7** [kangarko/CoreArena#284]
Please see your answer to #281. Sorry I think I phrased the question wrong and I don't understand your answer (and not sure how to reopen that case).

What I would like to know is:
Is the plugin coded so that the only way you can buy rewards (and have them not disappear), is when you are in-game (which is difficult because you get killed when you are in the menu.)? I just want to be sure I am not setting this up incorrectly. This is turning out to be a real problem on my servers which is why I want to make sure.

Thanks!

**Comment by Ekute-hub** [comment]
Hi, so if you want to sell item versus nuggets in arena you should use a sign with [upragde] on it,
if you try to give an item to a player to use it in arena, use kit tiers system or change a line in general config for force players to use there own stuff.
